cloudify_agent/installer/script.py

```diff
@@ -203,3 +207,27 @@ def install_script_path(cloudify_agent):
         yield script_path
     finally:
         os.remove(script_path)
+
+
+def _cleanup_after_installation(path):
+    """Mark path to be deleted after agent installation.
+
+    This simply adds the path to cloudify_agent inside runtime properties,
+    so that it can be removed later.
+    """
+    cloudify_agent = ctx.instance.runtime_properties.get('cloudify_agent', {})
```

Comment:
This seems unnecessary - you could just pass `self.cloudify_agent` from the above class